src/form-elements/FormElementCompliance.tsx

```diff
@@ -0,0 +1,224 @@
+import * as React from 'react'
+import clsx from 'clsx'
+
+import FormElementOptions from '../components/FormElementOptions'
+import useFormElementOptions from '../hooks/useFormElementOptions'
+import useBooleanState from '../hooks/useBooleanState'
+import useToggleComplianceChildren from '../hooks/useToggleComplianceChildren'
+import { FormTypes } from '@oneblink/types'
+import OptionButton from './OptionButton'
+import FormElementLabelContainer from '../components/FormElementLabelContainer'
+import ComplianceButton from './ComplianceButton'
+import FormElementFiles, { FilesElementFile } from './FormElementFiles'
+import FormElementTextarea from './FormElementTextarea'
+
+interface Props {
+  id: string
+  element: FormTypes.ComplianceElement
+  value: unknown
+  onChange: (
+    formElement: FormTypes.FormElement,
+    newValue: Value | undefined,
+  ) => unknown
+  onConditionallyShowOption: (
+    choiceElementOption: FormTypes.ChoiceElementOption,
+  ) => boolean
+  displayValidationMessage: boolean
+  validationMessage: string | undefined
+  isEven?: boolean
+}
+
+interface Value {
+  value: unknown
```

Comment:
```suggestion
  value: string
```

Reply:
This would require changing a lot more, including the hook that radeio buttons also use. Also, it can be an array as well

Reply:
it can be an array?

Reply:
Sorry, no it can't, but the hook that is used for radios (and checkboxes) can spit the value out as an array, so it uses unknown